src/service/viewer-impl.js

```diff
@@ -501,6 +506,14 @@ export class Viewer {
     return capabilities.split(',').indexOf(name) != -1;
   }
 
+  /**
+   * Whether the viewer can render templates.
+   * @return {boolean}
+   */
+  canRenderTemplates() {
```

Comment:
Since we moved this here, we should probably call this now `isSupported()` or similar.

Reply:
Yah, update everything that knows about the string `'viewerRenderTemplate'`.